wherehows-backend/conf/application.conf

```diff
@@ -75,6 +75,7 @@ hikaricp.dialect = ${?WHZ_DB_DIALECT}
 scheduler.check.interval = 5
 scheduler.check.interval = ${?WHZ_SCHEDULER_CHECK_INTERVAL}
 
+etl.java.cmd = "java";
```

Comment:
What's the alternative values since we made this a variable?

Reply:
For internal override